apps/apputil.hpp

```diff
@@ -221,9 +247,16 @@ inline options_t ProcessOptions(char* const* argv, int argc, std::vector<OptionS
     {
         const char* a = *p;
         // cout << "*D ARG: '" << a << "'\n";
-        if ( moreoptions && a[0] == '-' )
+        if (moreoptions && a[0] == '-')
         {
-            current_key = a+1;
+            string key(a + 1);  // omit '-'
+            size_t pos = key.find_first_of(":=");
```

Comment:
Using `=` as an alternative value specification is rather not a good idea, especially if the argument might contain spaces. Note that we handle this `:` only for backward compatibility, the desired way to specify values for options is to use the next argument.

Reply:
Deleted `=`